### DISCUSSION TITLE
Windows 11 use clic right context menu Windows 10

### DISCUSSION THREAD

**Discussion by AMJK** [LeDragoX/Win-Debloat-Tools#115] [Ideas]
Hello, 
Do you think you can add option for activate old context menu to right click on Windows 11.
I hate need click on "More Option".

This script can change for use old context :

`reg.exe add "HKCU\Software\Classes\CLSID\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2}\InprocServer32" /f`
`taskkill /f /im explorer.exe`
`start explorer.exe`

Thank you so much

**Comment by LeDragoX** [comment]
I've already done that on _"develop"_ branch, but didn't merged yet cause i need to find more stuff to remove.
